ci/cron/daily-compat.yml

```diff
@@ -173,6 +173,94 @@ jobs:
+          source $(bash_lib)
+
+          docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD"
+          IMAGE=$(cat ci/oracle_image)
+          docker pull $IMAGE
+          # Cleanup stray containers that might still be running from
+          # another build that didn’t get shut down cleanly.
+          docker rm -f oracle || true
+          # Oracle does not like if you connect to it via localhost if it’s running in the container.
+          # Interestingly it works if you use the external IP of the host so the issue is
+          # not the host it is listening on (it claims for that to be 0.0.0.0).
+          # --network host is a cheap escape hatch for this.
+          docker run -d --rm --name oracle --network host -e ORACLE_PWD=$ORACLE_PWD $IMAGE
+          function cleanup() {
+              docker rm -f oracle
+          }
+          trap cleanup EXIT
+          testConnection() {
+              docker exec oracle bash -c 'sqlplus -L '"$ORACLE_USERNAME"'/'"$ORACLE_PWD"'@//localhost:'"$ORACLE_PORT"'/ORCLPDB1 <<< "select * from dba_users;"; exit $?' >/dev/null
+          }
+
+          # dont want to wait forever to test connection , 15m is more than sufficient here.
+          declare -fxr testConnection
+          timeout 15m bash -c 'until testConnection; do echo "Could not connect to Oracle, trying again..." ; sleep 1 ; done'
+
+          # Actually run some tests
+
+          bazel build //ledger-service/http-json-perf/...
+          DAR="${PWD}/bazel-bin/ledger-service/http-json-perf/LargeAcs.dar"
+          JWT="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJNeUxlZGdlciIsImFwcGxpY2F0aW9uSWQiOiJmb29iYXIiLCJhY3RBcyI6WyJBbGljZSJdfX0.VdDI96mw5hrfM5ZNxLyetSVwcD7XtLT4dIdHIOa9lcU"
+
+          TEST_CASES="\
+           populateCache \
+           fetchByKey \
+           fetchByQuery \
+          "
+
+          START=$(git log -n1 --format=%cd --date=format:%Y%m%d).$(git rev-list --count HEAD).$(Build.BuildId).$(git log -n1 --format=%h --abbrev=8)
+          REPORT_ID="http_json_perf_oracle_results_${START}"
+          OUT="$(Build.StagingDirectory)/${REPORT_ID}"
+
+          # Overall we want to populate the test cache , run some fetch by key queries and then some index queries
+
+          export NUM_RECORDS=100000 # 100k ACS for a single template
+          export NUM_QUERIES=10000 # 10k queries in total
+          export NUM_READERS=100 # 100 users in parallel.
+
+          for CASE in $TEST_CASES; do
+            RUN_MODE=${CASE} USE_DEFAULT_USER=true RETAIN_DATA=true bazel run //ledger-service/http-json-perf:http-json-perf-binary-ee -- \
+              --scenario=com.daml.http.perf.scenario.OracleMultiUserQueryScenario \
+              --dars=${DAR} \
+              --reports-dir=${OUT}_${CASE} \
+              --jwt=${JWT} \
+              --query-store-index=oracle
+          done
+
+          GZIP=-9 tar -zcvf ${OUT}.tgz ${OUT}_*
```

Comment:
Do we need to be verbose here?

Reply:
Also `${OUT}_*` looks like a list of files. I personally like my tar files to be contained in a folder; could you maybe do `--reports=${OUT}/${CASE}` instead of `${OUT}_${CASE}`?